public/pages/Dashboard/utils/constants.js

```diff
@@ -37,8 +37,8 @@ export const DEFAULT_GET_ALERTS_QUERY_PARAMS_BY_TRIGGER = {
 export const PPL_SEARCH_PATH = '_plugins/_ppl';
 export const DEFAULT_LOG_PATTERN_TOP_N = 3;
 export const DEFAULT_LOG_PATTERN_SAMPLE_SIZE = 20;
-export const DEFAULT_ACTIVE_ALERTS_TOP_N = 10;
-export const DEFAULT_DSL_QUERY_DATE_FORMAT = 'YYYY-MM-DDTHH:mm:ssZ'
+export const DEFAULT_ACTIVE_ALERTS_TOP_N = 1;
```

Comment:
This PR is related to AI features, but this setting is not only related to AI features. We should have a separate setting for AI features.

Reply:
this setting is introduced and used for AI features only, let's rename it to `DEFAULT_ACTIVE_ALERTS_TOP_N_AI` to make it more clear